packages/core/lib/utils.ts

```diff
@@ -21,54 +21,65 @@ const splitAttrs = (str) => {
   return res;
 };
 
+const defaultPlugins = [
+  "removeDoctype",
```

Comment:
Check preset-default for simpler config extending
https://github.com/svg/svgo/releases/tag/v2.4.0